bundle.Dockerfile

```diff
@@ -7,7 +7,7 @@ LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=sandboxed-containers-operator
 LABEL operators.operatorframework.io.bundle.channels.v1=preview-1.0
 LABEL operators.operatorframework.io.bundle.channel.default.v1=preview-1.0
-LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.12.0
+LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.11.0+git
```

Comment:
Why do you go down a minor version? That should probably be explained in the cover letter.

Reply:
This needs rebase.